libraries/helpers.rb

```diff
@@ -28,6 +28,7 @@ def psql_command_string(new_resource, query, grep_for: nil, value_only: false)
       cmd << " --host #{new_resource.host}" if new_resource.host
       cmd << " --port #{new_resource.port}" if new_resource.port
       cmd << ' --tuples-only'               if value_only
+      cmd << ' --no-psqlrc'
       cmd << " | grep #{grep_for}"          if grep_for
```

Comment:
I think this might resolve the cookstyle issue.
```suggestion
      cmd << " --no-psqlrc | grep #{grep_for}" if grep_for
```

Reply:
Thank you for the review.

I had the impression from the [CONTRIBUTING.md](https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD#code-review-process) that the changelog would be added at a later stage, during the review process 😅
My impression was wrong.

Reply:
Ah  yes, we probably need to amend that.